frontend/src/components/RadialBarChart/RadialBarChart.tsx

```diff
@@ -1,32 +0,0 @@
-import React, { Suspense } from 'react';
```

Comment:
why is this file deleted?

Reply:
deleted this component file because it wasn't used at all - think it showed up here since the data migration branch was not merged to the latest dev?

Reply:
wierd, i thought it was updated. I think we may have borked something then. No worries, we will just accept those changes then.